src/Client.js

```diff
@@ -213,7 +196,7 @@ export default class Client {
     return `${this.options.baseUrl}${endpoint}${query}`;
   }
 
-  buildSignedUrl(method, ...args) {
+  async buildSignedUrl(method, ...args) {
```

Comment:
This is going to be a major change. :smile:

Reply:
Yes, this will be released as 3.0.0.